src/cswinrt/strings/ComInteropHelpers.cs

```diff
@@ -485,3 +485,69 @@ public static DisplayInformation GetForMonitor(IntPtr monitor)
     }
 #endif
 }
+
+namespace Windows.ApplicationModel.DataTransfer
+{
+#if UAC_VERSION_1
+#if NET
+    [global::System.Runtime.Versioning.SupportedOSPlatform("Windows10.0.10240.0")]
+#endif
+#if EMBED
+    internal
+#else
+    public
+#endif
+    static class DataTransferManagerInterop
+    {
+        private static readonly global::System.Guid IDataTransferManagerInterop_IID = new global::System.Guid("3A3DCD6C-3EAB-43DC-BCDE-45671CE800C8");
+
+        private static readonly global::System.Guid riid = new global::System.Guid("A5CAEE9B-8708-49D1-8D36-67D25A8DA00C");
+
+        private static global::WinRT.FactoryObjectReference<global::WinRT.IObjectReference> factoryObjectReference;
+
+        static DataTransferManagerInterop()

```

Comment:
if possible, I would just do an inline variable declaration rather than in a static constructor.